__tests__/components/noscript_test.js

```diff
@@ -6,12 +6,6 @@ const { axe, toHaveNoViolations } = require("jest-axe");
 expect.extend(toHaveNoViolations);
 
 describe("Test Noscript component", () => {
-  it("displays a message inside a noscript tag", () => {
-    const noscript = mount(<Noscript t={key => key} />);
-
-    expect(noscript.find("div.copy p").text()).toEqual("noscript");
-  });
-
```

Comment:
had to delete this test because now noscript doesn't render on the client, only on the server